src/main/kotlin/com/openlattice/data/storage/partitions/PartitionManager.kt

```diff
@@ -175,7 +175,7 @@ class PartitionManager @JvmOverloads constructor(
         return BasePostgresIterable(
                 PreparedStatementHolderSupplier(hds, EMPTIEST_PARTITIONS) { ps ->
                     ps.setArray(1, PostgresArrays.createIntArray(ps.connection, partitionList))
-                    ps.setObject(2, desiredPartitions)
+                    ps.setInt(2, desiredPartitions)
```

Comment:
is desiredPartitions just a single partition?

Reply:
it is an `Int`

Reply:
:sadness:

~what if~ we rename it to a singular noun to reflect this fact

Reply:
`~then~` the world will be slightly better